modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java

```diff
@@ -254,6 +256,8 @@
 	public static final String DOWNLOAD_COMPILED_JSP_TASK_NAME =
 		"downloadCompiledJSP";
 
+	public static final String GENERATE_POM_INFO_TASK_NAME = "generatePomInfo";
```

Comment:
You added a new public member, did you run `gradlew baseline` to check it's not a minor change?

Reply:
I can use private instead.